pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -746,7 +757,9 @@ LogicalResult ConvertAtenOp<AtenSigmoidOp>::matchAndRewrite(
   Value negVal = rewriter.create<mhlo::NegOp>(loc, input);
   Value expVal = rewriter.create<mhlo::ExpOp>(loc, negVal);
   Value addVal = rewriter.create<mhlo::AddOp>(loc, expVal, one);
-  rewriter.replaceOpWithNewOp<mhlo::DivOp>(op, one, addVal);
+  Value output = rewriter.create<mhlo::DivOp>(loc, one, addVal);
+  rewriter.replaceOpWithNewOp<mhlo::ConvertOp>(
```

Comment:
what's the intention for such `ConvertOp` added at the end?

Reply:
Because we are dynamic shapes, some static shapes are not carried during conversion. But the latest llvm mhlo will check the output type before and after the conversion. We introduce ConvertOp to avoid the error.